adding viewer and editor to sign in docs (#2946)
chef/automate: dev-docs/DEV_ENVIRONMENT.md

# start_all_services
```

Open <https://a2-dev.test/> in your browser.

> Sign in as `admin`, `viewer`, or `editor` all with the same password of `chefautomate`.

Reviewer: ```suggestion
Sign in as `admin`, `viewer`, or `editor`, each using the same password of `chefautomate`.
```

  - `admin` gives access to everything.

>   - `editor` is what most users expereince which gives them access to everything except for `Node Credentials`, `Notifications`, Identity pages, Roles, and Policies.

Reviewer 2: ```suggestion
  - `editor` defines the permissions most users will have. It gives them access to everything except for `Node Credentials`, `Notifications`, Identity pages, Roles, and Policies.
```
